app/models/claim.rb

```diff
@@ -23,10 +23,17 @@ class Claim < ApplicationRecord
   belongs_to :school
   belongs_to :provider
 
-  has_many :mentor_trainings
-  has_many :mentors, through: :mentor_trainings
+  has_many(
+    :mentor_trainings,
+    -> { includes(:mentor).order("mentors.first_name", "mentors.last_name") },
+  )
+  has_many(
```

Comment:
does this need to be `dependent: :destroy`?

Reply:
Actually, I don't think we should add this. If we do, it would be easier to destroy claims in the console. Currently without this you cannot do `Claim.last.destroy`